backstop/tests/menu.html

```diff
@@ -442,7 +442,11 @@ <h2>Skeleton</h2>
           style="--iui-menu-item-content-skeleton-max-width: 30%;"
         >
           <span class="iui-content">
-            <div class="iui-menu-label iui-skeleton"></div>
+            <div
+              class="iui-menu-label iui-skeleton"
+              aria-hidden="true"
+            ></div>
+            <div class="iui-visually-hidden">Loading...</div>
```

Comment:
So in React you would have a component, which gonna deliver all of these elements and have prop for `loading` text?

Reply:
Yup

Reply:
`<MenuItemSkeleton hasIcon hasDescription />` something like that.